target/linux/x86/atom64/profiles/001-fw8756.mk

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2006-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+define Profile/fw8756
+  NAME:=fw8756
+  PACKAGES:=kmod-w83627-wdt kmod-hwmon-nct6775 lcdproc
```

Comment:
For this specific package inclusion, ``target/linux/x86/64/target.mk`` can be used.